docs/quickstarts/excel-quickstart-vue.md

```diff
@@ -42,164 +38,160 @@ Each add-in requires a manifest file to define its settings and capabilities.
-        height: 80px;
-        overflow: hidden;
-    }
-
-    #content-main {
-        background: #fff;
-        position: fixed;
-        top: 80px;
-        left: 0;
-        right: 0;
-        bottom: 0;
-        overflow: auto;
-    }
-
-    .padding {
-        padding: 15px;
-    }
-    </style>
-    ```
+1. Open the `public/index.html` file and add the following `<script>` tag immediately before the `</head>` tag.
+
+   ```html
+   <script src="https://appsforoffice.microsoft.com/lib/1/hosted/office.js"></script>
+   ```
+
+2. Replace the contents of `src/main.js` with the following code:
+
+   ```js
+   import Vue from 'vue'
+   import App from './App.vue'
+
+   Vue.config.productionTip = false
+
+   window.Office.initialize = () => {
+     new Vue({
+       render: h => h(App),
+     }).$mount('#app')
+   }
+   ```
+
+3. Open `src/App.vue`, replace file contents with the following code.
+
+   ```html
+   <template>
+     <div id="app">
+       <div class="content">
+         <div class="content-header">
+           <div class="padding">
+             <h1>Welcome</h1>
+           </div>
+         </div>
+         <div id="content-main">
+           <div class="padding">
+             <p>
+               Choose the button below to set the color of the selected range to
+               green.
+             </p>
+             <br />
+             <h3>Try it out</h3>
+             <button @click="onSetColor">Set color</button>
+           </div>
+         </div>
+       </div>
+     </div>
+   </template>
+
+   <script>
+     export default {
+       name: 'App',
+       methods: {
+         onSetColor() {
+           window.Excel.run(async context => {
+             const range = context.workbook.getSelectedRange()
+             range.format.fill.color = 'green'
+             await context.sync()
+           })
+         },
+       },
+     }
+   </script>
+
+   <style>
+     .content-header {
+       background: #2a8dd4;
+       color: #fff;
+       position: absolute;
+       top: 0;
+       left: 0;
+       width: 100%;
+       height: 80px;
+       overflow: hidden;
+     }
+
+     .content-main {
+       background: #fff;
+       position: fixed;
+       top: 80px;
+       left: 0;
+       right: 0;
+       bottom: 0;
+       overflow: auto;
+     }
+
+     .padding {
+       padding: 15px;
+     }
+   </style>
+   ```
 
 ## Start the dev server
 
 1. From the terminal, run the following command to start the dev server.
 
-    ```command&nbsp;line
-    npm start
-    ```
+   ```command&nbsp;line
+   npm run serve
+   ```
 
-2. In a web browser, navigate to `https://localhost:8080`. If your browser indicates that the site's certificate is not trusted, you will need to configure your computer to trust the certificate. 
+2. In a web browser, navigate to `https://localhost:3000` (notice the `https`). If your browser indicates that the site's certificate is not trusted, you will need to configure your computer to trust the certificate.
```

Comment:
What is the reason to link to `OfficeDev/generator-office:src/docs/ssl.md`? 
Since we are using webpack-dev-server and not the server provided by yeoman I don't think it would be helpful.